modules/file-system/pre-existing-network-storage/README.md

```diff
@@ -24,6 +24,22 @@ This creates a pre-existing-network-storage module in terraform at the
 provided IP in `server_ip` of type nfs that will be mounted at `/home`. Note
 that the `server_ip` must be known before deployment.
 
+### Mounting
+
+For some `fs_type`, this module will provide `client_install_runner` and
+`mount_runner` outputs. These can be used to create a startup script to mount
+the network storage system.
+
+Supported `fs_type`:
+
+- lustre (DDN)
+
+[scripts/mount.sh](./scripts/mount.sh) is used as the contents of
+`mount_runner`. This script will update `/etc/fstab` and mount the
+network storage. Behavior is undefined if:
+
+- System already has an entry in `/etc/fstab` for `local_mount`.
```

Comment:
this is not an fs type, I think it is confusing to have it this way.

Reply:
Case 1: remote mount is different and local mount is the same from existing fstab entry:
- entry will be added to fstab
- the device will never be mounted as the local mount will already be busy
- even if the local mount point doesn't have anything mounted, when you call mount target it will select the first matching item in the list, so your device will never be mounted

Case 2: remote mount is same and local mount is different from existing fstab entry:
- entry will be added to fstab
- device will mount normally (you will have the same device mounted in 2 locations)

So how ever we say it the real problem arises if local mount is the same, which is what I am trying to call out here. Maybe I should change the `MOUNT_IDENTIFIER` to only look for common local mount. Basically it doesn't matter if remote mount is different, if there is already an entry for that local mount then we error. WDYT?

Then I could change the wording to say, "mounting will fail if there is an entry in fstab that already uses the same local mount". This is a much clearer and definitive statement.

Reply:
Done. The wording has been updated.